microsoft/testsuites/performance/nvmeperf.py

```diff
@@ -37,6 +39,25 @@ class NvmePerformace(TestSuite):
         ),
     )
     def perf_nvme(self, node: Node, result: TestResult) -> None:
+        self._perf_nvme(node, IoEngine.LIBAIO, result)
+
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test NVMe disk performance
+        using 'io_uring' as ioengine.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            supported_features=[Nvme],
+        ),
+    )
```

Comment:
for perf_nvme, it has requirement for count of nvme disks, for perf_nvme_io_uring, by default it uses 1 nvme disk, is it expected?

Reply:
There are new VMs with just 1 NVMe. So its a valid case.